index.html

```diff
@@ -471,13 +471,16 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         through a transformation function (<b>iteratee</b>). The <tt>iteratee</tt>
         is passed three arguments: the <tt>value</tt>, then the <tt>index</tt>
         (or <tt>key</tt>) of the iteration, and finally a reference to the entire
-        <tt>list</tt>.
+        <tt>list</tt>. Selective <b>Underscore</b> functions marked as <tt>guarded</tt>
+        can be used as an <tt>iteratee</tt> to <b>_.map</b>.
```

Comment:
@jdalton The annotated source code makes it explicit that these functions are guarded specifically for _.map. There is no mention anywhere of _.filter, _.every, or any others, being used with guarded functions.

You can pass guarded functions into methods like _.filter, but it will not produce the expected results:
![screen shot 2015-02-09 at 11 54 18 am](https://cloud.githubusercontent.com/assets/5640348/6114399/b326f952-b052-11e4-9835-750b82479157.png)